admin/user-management.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "User Management"
+description: Learn how to add and delete Coder users.
+---
+
+Site managers can create and manage users via the User Management dashboard,
+which is accessible under **Manage** > **Users**.
```

Comment:
What do you think of the following:

```txt
[Site managers](link-to-roles) can create and manage users from the users page of the dashboard
which is accessible under **Manage** > **Users**.
```

Rationale:

- opportunity to link to rbac roles
- the capital casing of `User Manage` followed by lowercase dashboard feels a bit out of place, and we don't call it that in-app
- it's really a page within the dashboard, as opposed to a separate dashboard (current wording makes it feel like a separate app)

Not gonna die on any of these hills, but thought I'd suggest it while we're here